scheduler/scheduler-server/src/test/java/functionaltests/recover/RunningTaskRecoveryWithInProcessTaskExecutorTest.java

```diff
@@ -34,11 +34,11 @@
  * @author ActiveEon Team
  * @since 20/09/17
  */
-public class TaskReconnectionWithInProcessTaskExecutorTest extends TaskReconnectionToRecoveredNodeTest {
+public class RunningTaskRecoveryWithInProcessTaskExecutorTest extends RunningTaskRecoveryWithRecoveredNodeTestBase {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2187)